### PR TITLE
User Guide modifications

### PR DESCRIPTION
**Pull request by jweeyh**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -257,15 +246,22 @@ An employee can have any number of tags (including 0)
...
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add id/123 n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Residences p/1234567 t/intern t/vegan`
+* `add id/777 n/John Doe p/98765432 a/311, Clementi Ave 2, #02-25 e/johnd@example.com t/vegan t/intern`
+
+
+![result for 'add id/777 n/John Doe p/98765432 a/311, Clementi Ave 2, #02-25 e/johnd@example.com t/vegan t/intern'](images/UiAddEmployeeCommand.png)
+
 
 ### 7.1.2. Listing all employees : `list`
```
Comment (kennethk-1201) [inline docs/UserGuide.md:256]: Need to update the numbering from 7 to 5

**Review by kennethk-1201 — CHANGES_REQUESTED** [review]
Can you shift the ldhc command from the end of the department section to the end of the leave section, and also in the DG as well. (flows better)

**Review by kennethk-1201 — APPROVED** [review]
LGTM

**Review by 4ndrelim — APPROVED** [review]
LGTM

edit: oops didn't see kiang reviewed agn

**Pull request merged**